openfisca_us/variables/usda/usda.py

```diff
@@ -0,0 +1,15 @@
+from openfisca_core.model_api import *
+from openfisca_us.entities import *
+from openfisca_us.tools.general import *
+
+
+class is_usda_elderly(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = YEAR
+    documentation = "Is elderly per USDA guidelines"
+    label = ""
+
+    def formula(persons, period, parameters):
+        min_age = parameters(period).usda.elderly.values.age
```

Comment:
Let's rename `elderly.yaml` to `elderly_age.yaml` to describe that it's the parameter, then this is all you need:
```suggestion
        min_age = parameters(period).usda.elderly_age
```